src/test/java/org/testcontainers/cloud/turbomode/sample/TestcontainersCloud1Test.java

```diff
@@ -1,20 +1,11 @@
 package org.testcontainers.cloud.turbomode.sample;
 
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-import org.testcontainers.containers.KafkaContainer;
-import org.testcontainers.junit.jupiter.Container;
-import org.testcontainers.junit.jupiter.Testcontainers;
-import org.testcontainers.utility.DockerImageName;
 
-@Testcontainers
-public class TestcontainersCloud1Test {
-
-    @Container
-    private KafkaContainer kafka = new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:6.2.1"));
+public class TestcontainersCloud1Test extends AbstractIntegrationTest {
 
     @Test
-    public void testFindingAnInsertedValue() {
-        Assertions.assertNotNull(kafka.getBootstrapServers());
+    public void writeAndVerify() {
```

Comment:
Absolute wincy-tincy-minor nit: my understanding is that the preferred syntax for tests using the new junit is to leave the modifier (`public` in this case) out completely (so just `void writeAndVerify()` here? But obviously it works as is, so entirely optional.